src/main/cli/settings.c

```diff
@@ -705,7 +705,7 @@ const clivalue_t valueTable[] = {
 #if defined(USE_GYRO_SPI_ICM20649)
     { "acc_high_range",             VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_ACCELEROMETER_CONFIG, offsetof(accelerometerConfig_t, acc_high_fsr) },
 #endif
-    { PARAM_NAME_ACC_LPF_HZ,        VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 0, 400 }, PG_ACCELEROMETER_CONFIG, offsetof(accelerometerConfig_t, acc_lpf_hz) },
+    { PARAM_NAME_ACC_LPF_HZ,        VAR_UINT16 | MASTER_VALUE, .config.minmaxUnsigned = { 0, 250 }, PG_ACCELEROMETER_CONFIG, offsetof(accelerometerConfig_t, acc_lpf_hz) },
```

Comment:
if 250 is the new max change from a uint_16 to a uint_8

Reply:
Good point. I am still not 100 % sure about max. So we can leave this open and change it once we decide.

I think it should be 250Hz as data is used in ATTITUDE/IMU task which runs at 500Hz freq.

Reply:
Set max to 500 as max IMU/ATTITUDE task is slightly above 1000Hz and we need to max half freq. (nyquist)

Reply:
> Set max to 500 as max IMU/ATTITUDE task is slightly above 1000Hz and we need max half freq. (nyquist)

Well, it means that 500 is a max (assuming you are using a biquad, pt filters are actually still fine even when higher.), but this doesn't mean that you should use a value that high.

Reply:
I am using PT2 here. We cannot go higher than 500Hz if IMU task runs at 1000Hz.

Reply:
You can go past 500hz without bad side effects. Pt2 is just stacked pt1, and technically pt2 at 500hz is 2 pt1 above 500hz. Pt filters dont suffer issues when above nyquist limit. Biquad filters do have issues past nyquist limit though.

Reply:
@daleckystepan is using the Nyquist–Shannon sampling theorem which states that you cannot go past half the new sampling rate if you want to guarantee that there will be no significant aliasing. That's why for downsampling from 8000Hz to 1000Hz we need the cutoff at 500Hz minimum to satisfy the theorem and guarantee that aliasing is dealt with.

Reply:
(@KarateBrot : Actually you need to 'remove' all frequencies above 500Hz. Filter at 500Hz will (by definition) reduce signal by 3dB (50%) at 500Hz. )

Reply:
Yeah, fair enough. That's why we often went with PT2's here to get a bit more attenuation above cutoff than with a PT1.